sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Catalog.scala

```diff
@@ -41,6 +41,10 @@ trait Catalog {
     throw new UnsupportedOperationException
   }
 
+  def runNativeCommand(sql: String): Seq[Row] = {
```

Comment:
Running commands shouldn't be part of the catalog interface. This is only used in 1 place anyway so I wouldn't bother making an abstraction here.